go/ros/bag2mcap.go

```diff
@@ -225,7 +232,7 @@ func channelIDForConnection(connID uint32) (uint16, error) {
 	return uint16(connID), nil
 }
 
-func Bag2MCAP(w io.Writer, r io.Reader, opts *mcap.WriterOptions) error {
+func Bag2MCAP(w io.Writer, r io.Reader, opts *mcap.WriterOptions, progressBar *progressbar.ProgressBar) error {
```

Comment:
I feel like i should just put the progress bar in this function instead of injecting it 🤷

Reply:
that does seem nicer, but you'd just need to pass in a messageCount.

Maybe the cleanest option would be to pass in the `*Info` struct, and then construct the progress bar in here? You need to call Info() outside this function (as you do) or else you would need to pass this a ReadSeeker, which you're not in position to do.